org.osgi.test.bom/pom.xml

```diff
@@ -1,115 +1,87 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-	* Copyright (c) OSGi Alliance (2021). All Rights Reserved.
-	*
-	* Licensed under the Apache License, Version 2.0 (the "License");
-	* you may not use this file except in compliance with the License.
-	* You may obtain a copy of the License at
-	*
-	* http://www.apache.org/licenses/LICENSE-2.0
-	*
-	* Unless required by applicable law or agreed to in writing, software
-	* distributed under the License is distributed on an "AS IS" BASIS,
-	* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-	* See the License for the specific language governing permissions and
-	* limitations under the License.
+	Copyright (c) Contributors to the Eclipse Foundation
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+	SPDX-License-Identifier: Apache-2.0
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>org.osgi</groupId>
+
+	<parent>
+		<groupId>org.osgi</groupId>
+		<artifactId>org.osgi.test.parent</artifactId>
+		<version>${revision}</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
 	<artifactId>org.osgi.test.bom</artifactId>
-	<version>${revision}</version>
-	<packaging>pom</packaging>
-	<name>OSGi-Test Bill of Materials</name>
 	<description>OSGi-Test Bill of Materials</description>
+	<name>${project.groupId}:${project.artifactId}</name>
+	<url>https://www.osgi.org</url>
+	<scm>
+		<url>https://github.com/osgi/osgi-test</url>
+		<connection>scm:git:https://github.com/osgi/osgi-test.git</connection>
+		<developerConnection>scm:git:git@github.com:osgi/osgi-test.git</developerConnection>
+	</scm>
+
+	<packaging>pom</packaging>
+
+	<properties>
+		<bnd.resolve.skip>true</bnd.resolve.skip>
```

Comment:
I am concerned that while this is necessary to build in our build, this will bleed through into all users of the bom? Or is that an incorrect assumption?

Reply:
I'll check

Reply:
In my test it did not appear to have any side effect. And I don't believe properties of the Bom are ever consumed downstream from a bom.

Reply:
However if there is real concern we can directly configure the plugin instead of using the property.